src/test/java/step1/SetTest.java

```diff
@@ -0,0 +1,69 @@
+
+package step1;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+	private Set<Integer> numbers;
+
+	@BeforeEach
+	void setUp() {
+		numbers = new HashSet<>();
+		numbers.add(1);
+		numbers.add(1);
+		numbers.add(2);
+		numbers.add(3);
+	}
+
+	@Test
+	@DisplayName("Set 크기 확인")
+	public void 요구사항1() throws Exception {
+		assertThat(numbers.size()).isEqualTo(3);
+		assertThat(numbers).hasSize(3);
```

Comment:
검증에 대한 내용이 같아서 하나의 검증 코드만 있어도 괜찮을 것 같아요

```suggestion
		assertThat(numbers).hasSize(3);
```